src/gl-state-egl.cpp

```diff
@@ -303,11 +318,23 @@ GLStateEGL::~GLStateEGL()
 
     if(!eglReleaseThread())
        Log::error("eglReleaseThread failed\n");
+
+   close_library(egl_library_);
```

Comment:
this looks weirdly indented (unless it's just my browser)

Reply:
Fixed. It wasn't just you.